docs/modules/ROOT/pages/reference/extensions/fhir.adoc

```diff
@@ -62,7 +62,19 @@ By default, only FHIR versions `R4` & `DSTU3` are enabled in native mode, since
 
 Enable FHIR DSTU2 Specs in native mode.
 | `boolean`
-| `true`
+| `false`
```

Comment:
Do we need some migration guide for this one ?

Reply:
Yeah, I can add some notes about the change to the config defaults.

Reply:
I pushed a commit with a migration guide for 2.8.0.

Reply:
Awesome